proto/viam/app/data/v1/data.proto

```diff
@@ -54,6 +54,9 @@ service DataService {
 
   // BoundingBoxLabelsByFilter gets all string labels for bounding boxes from data based on given filter.
   rpc BoundingBoxLabelsByFilter(BoundingBoxLabelsByFilterRequest) returns (BoundingBoxLabelsByFilterResponse);
+
+  // GetDatabaseConnection gets a hostname connection for a Viam user who is an org owner to access a MongoDB Atlas Data Federation instance.
```

Comment:
Is it good practice to say "for a Viam user who is an org owner to access..." or to leave it as "for any Viam user to access..."? The reason I ask is we're somewhat explaining our perms by specifying only an org owner can get an ADF connection and I'm not sure if that's a security concern.